cmd/configure.ac

```diff
@@ -22,11 +22,7 @@ AC_SYS_LARGEFILE
 # Checks for header files.
 AC_CHECK_HEADERS([fcntl.h limits.h stdlib.h string.h sys/mount.h unistd.h])
 AC_CHECK_HEADERS([sys/quota.h], [], [AC_MSG_ERROR(sys/quota.h unavailable)])
-AC_CHECK_HEADERS([xfs/xqm.h], [], [AC_MSG_ERROR(xfs/xqm.h unavailable)],
-[[#define _GNU_SOURCE
-#define _FILE_OFFSET_BITS 64
```

Comment:
out of curiosity, what macro defines this already?

Reply:
`AC_LANG_CONFTEST` will prepend `confdefs.h`. All prior `AC_DEFINE*` will add into `confdefs.h`.  `AC_SYS_LARGEFILE` will do and `AC_DEFINE` for `_FILE_OFFSET_BITS`. `AC_USE_SYSTEM_EXTENSIONS` will do one for `_GNU_SOURCE`.